src/helpers/translate.ts

```diff
@@ -28,11 +30,24 @@ export function t(input: string, params?: ParamsObject): string {
   }
 }
 
+interface ResultsArray<T = unknown> extends Array<T> {
+  toElement(): ReactNode;
+}
+
+let toElement = (elements: Array<ReactNode>) =>
+  createElement(Fragment, null, ...elements);
+
+let createResultsArray = (): ResultsArray => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let results: any = [];
+  results.toElement = () => toElement(results);
```

Comment:
Oops, this line broke the test.

Probably need to find non hacky solution

<img width="791" alt="image" src="https://user-images.githubusercontent.com/19742419/77471200-72bc5c80-6e44-11ea-9b28-1885d8d9995a.png">